app/models/review.rb

```diff
@@ -0,0 +1,5 @@
+class Review < ApplicationRecord
+  validates :content, presence: true
+  belongs_to :user
```

Comment:
add a ", dependent: :destroy" to user and to bar, also there should be a validation presence true for stars

Reply:
@ShadyaJ I have updated the PR, please review. Thanks.